lib/Master.php

```diff
@@ -112,7 +112,9 @@ public function handleLoginRequest($param) {
 			return;
 		}
 
-		$options = [];
+		$target = $this->request->getPathInfo() === false ? '/' : '/index.php' . $this->request->getPathInfo();
```

Comment:
I do think you'll need to check if pathInfo does not yet start with index.php (else you get `index.php/index.php/...`

Reply:
pathinfo always starts after index.php, tested on my system.

Reply:
Ah of course :+1: